chore: do not send hardhat_setLedgerOutputEnabled over http (#5578)
NomicFoundation/hardhat: .changeset/shaggy-bags-work.md

---
"hardhat": patch
---

> Do not send `http_setLedgerOutputEnabled` messages if they reacht the HTTP Provider to prevent unwanted warnings in the local hardhat node logs

Reviewer: typo 😅

Author: Good catch, thanks 🙇 

```suggestion
Do not send `http_setLedgerOutputEnabled` messages beyond the HTTP Provider to prevent unwanted warnings in the logs of the local hardhat node
```
